setup.py

```diff
@@ -45,6 +45,8 @@
             "wheel",
             # For `make package-upload`
             "twine",
+            # pydocstyle specific version till bug fixed
```

Comment:
I would link to the bug in question to make it easier to track when this can be changed

Reply:
Added :)